core/src/main/java/io/confluent/rest/SslFactory.java

```diff
@@ -159,6 +157,27 @@ public static SslContextFactory createSslContextFactory(SslConfig sslConfig) {
     return sslContextFactory;
   }
 
+  private static void configureSpiffeSslContext(
+      SslContextFactory.Server sslContextFactory,
+      X509Source x509Source,
+      boolean isMtls) {
+    SpiffeSslContextFactory.SslContextOptions options = SpiffeSslContextFactory.SslContextOptions
+        .builder()
+        .x509Source(x509Source)
+        .acceptAnySpiffeId()
```

Comment:
Right now, I configured to `acceptAnySpiffeId` . The authorization implementation is not part of this PR, it can be done in the following PR.

Reply:
Can we not configure a regex pattern here

Reply:
If you look at the native library [here](https://github.com/spiffe/java-spiffe/blob/a6059bdaebccfc118870893aee96af1fa028d991/java-spiffe-provider/src/main/java/io/spiffe/provider/SpiffeSslContextFactory.java#L150), they don't natively support regex. If we want to have more complex pattern, we have to define one more layer abstraction to do this functionality, which is why it won't be this PR scope.